src/goto-harness/recursive_initialization.cpp

```diff
@@ -37,7 +38,16 @@ void recursive_initializationt::initialize(
   }
   else if(type.id() == ID_pointer)
   {
-    initialize_pointer(lhs, depth, known_tags, body);
+    if(
```

Comment:
This could have been solved more elegantly if we used a handler chain pattern as the test-gen team does for their harness generation. Didn't want to do this atm because this was mostly about porting existing functionality, but might be worth converting this at some point.

Reply:
This is just a note, we don't need to do it right now.